src/components/side-panel/layer-panel/custom-palette.js

```diff
@@ -0,0 +1,284 @@
+    }
+
+    .sortableColors {
+      color: #fff;
+    }
+  }
+`;
+
+const StyledDragHandle = styled.div`
+  display: flex;
+  align-items: center;
+  opacity: 0;
+  z-index: 1000;
+  color: #fff
+  :hover {
+    cursor: move;
+    opacity: 1;
+    color: ${props => props.theme.textColorHl};
+  }
+`;
+
+const StyledTrash = styled.div`
+  .trashbin {
+    color: #fff;
+  }
+  height: 12px;
+  margin-left: auto;
+  margin-right: 12px;
+  :hover {
+    cursor: pointer;
+  }
+`;
+
+const StyledLine = styled.div`
+  width: 220px;
+  height: 1px;
+  background-color: #6a7485;
+  margin-top: 8px;
+  margin-left: 8px;
+`;
+
+const StyledSwatch = styled.div`
+  background-color: ${props => props.color};
+  width: 32px;
+  height: 18px;
+  display: inline-block;
+  cursor: pointer;
+`;
+
+const StyledColorRange = styled.div`
+  padding: 0 8px;
+  :hover {
+    background-color: ${props => props.theme.panelBackgroundHover};
+    cursor: pointer;
+  }
+`;
+
+const StyledButtonContainer = styled.div`
+  margin-top: 11px;
+  display: flex;
+  direction: rtl;
+`;
+
+const StyledButton = styled(Button)`
+  background-color: transparent;
+  color: #A0A7B4
+  :hover {
+    background-color: transparent;
+    color: #fff;
+  }
+`;
+
+const customStyles = {
+  content: {
+    top: '30%',
+    left: '340px',
+    right: 'auto',
+    bottom: 'auto',
+    padding: '0px 0px 0px 0px',
+    zIndex: 9999
+  }
+};
+
+const SortableItem = sortableElement(({children}) => (
+  <StyledSortableItem>{children}</StyledSortableItem>
+));
+
+const SortableContainer = sortableContainer(({children}) => (
+  <div>{children}</div>
+));
+
+const DragHandle = sortableHandle(({className, children}) => (
+  <StyledDragHandle className={className}>{children}</StyledDragHandle>
+));
+
+class CustomPalette extends Component {
+  static propTypes = {
+    customPalette: PropTypes.shape({
+      name: PropTypes.string,
+      type: PropTypes.string,
+      category: PropTypes.string,
+      colors: PropTypes.arrayOf(PropTypes.string)
+    }),
+    setCustomPalette: PropTypes.func,
+    showSketcher: PropTypes.bool,
+    onToggleSketcherUpdater: PropTypes.func
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentSwatchIndex: null
+    };
+  }
+
+  _setCustomPalette(colors) {
+    this.props.setCustomPalette({
+      name: 'Custom Palette',
+      type: null,
+      category: 'Uber',
+      colors
+    });
+  }
+  _onColorUpdate = color => {
+    const {colors} = this.props.customPalette;
+    const newColors = [...colors];
+    newColors[this.state.currentSwatchIndex] = color.hex;
+    this._setCustomPalette(newColors);
+  };
+
+  _onColorDelete = index => {
+    const {colors} = this.props.customPalette;
+    const newColors = [...colors];
+    newColors.splice(index, 1);
+    this._setCustomPalette(newColors);
+  };
+
+  _onColorAdd = () => {
+    const {colors} = this.props.customPalette;
+    const newColors = [...colors];
+    newColors.push('#F0F0F0');
```

Comment:
define a constant for this color outside your class component. Constants are usually defined at the top of the file after import statements:

```
const DEFAULT_NEW_COLOR = '#F0F0F0';
```
and your code will become:
```
newColors.push(DEFAULT_NEW_COLOR);
```